tests/test_extend_schema.py

```diff
@@ -48,6 +48,13 @@ class QuerySerializer(serializers.Serializer):
     )
 
 
+class MultipleChoiseSerializer(serializers.Serializer):
+    order_by = serializers.MultipleChoiceField(
```

Comment:
could you remove that serializer and move the field here? https://github.com/tfranzel/drf-spectacular/blob/7e77bb3ea852f407f1780c8ae6b5b17a9c8b8520/tests/test_extend_schema.py#L44

one less serializer in that test and it covers both exploded and nested.

Reply:
Done!